packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -8,17 +22,38 @@ There are two features which can be used to work around this size limit, batch c
 
 By default, the runtime is configured with a max batch size of `716800` bytes, which is lower than the 1MB limit. The reason for the lower value is to account for possible overhead from the op envelope and metadata.
 
-## Table of contents
+### How batching works
 
--   [Introduction](#introduction)
-    -   [Compression](#compression)
-    -   [Grouped batching](#grouped-batching)
-        -   [Risks](#risks)
-    -   [Chunking for compression](#chunking-for-compression)
-    -   [Disabling in case of emergency](#disabling-in-case-of-emergency)
-    -   [Example configs](#example-configs)
-    -   [How it works](#how-it-works)
-    -   [How grouped batching works](#how-grouped-batching-works)
+Batching in the context of Fluid ops is a way in which the framework groups ops and sends them to the server in a single payload. Afterwards the ops will be broadcasted in the same order to all the other connected clients. Additional logic and validation ensure that batches do not interleave and they are processed in isolation. This means two things:
```

Comment:
I'd more clearly separate Fluid promises to customers and service interaction. From Fluid perspective we should not speak how we send ops to service, as that's internal implementation (and in fact is being changed by your work). I'd only talk about - we promise that all side-effects (in the form of ops) that were done in one JS turn will apply without interruption in one go on all remote machines, and that such changes will not be interleaved by changes from other clients. I'd stress that's important correctness feature, as it allows remote machines not to expose partial state of data model (while rendering or processing user input).